concrete/controllers/panel/detail/page/composer.php

```diff
@@ -60,6 +60,21 @@ public function saveAndExit()
         }
     }
 
+    public function justSave()
+    {
+        if ($this->validateAction()) {
+            $r = $this->save();
```

Comment:
I copied this code from `saveAndExit` method. Should I update `saveAndExit` either?